user_sync/rules.py

```diff
@@ -408,6 +415,20 @@ def process_umapi_users(self, umapi_connectors):
                 self.logger.critical("Shouldn't happen! In secondary umapi %s, the following users were not found: %s",
                                      umapi_name, secondary_updates_by_user_key.keys())
 
+    def push_umapi_users(self, umapi_connectors):
+        """
+        This is where we push directory users to the Adobe side "as is".
+        :type umapi_connectors: UmapiConnectors
+        """
+        if umapi_connectors.get_secondary_connectors():
+            self.logger.debug('Pushing users to primary umapi...')
```

Comment:
This is a debug log message and I intentionally chose "umapi" lower-case so that it will align with the lower-case in the config file and also the other debug logging messages where "umapi" (lower-case) is part of the log string.  So I'm not willing to change this one.